test/scenarios-periodics/el93-crel@el94-src@optional-upgrade.sh

```diff
@@ -3,14 +3,16 @@
 # Sourced from scenario.sh and uses functions defined there.
 
 scenario_create_vms() {
-	prepare_kickstart host1 kickstart.ks.template rhel-9.3-microshift-source
-	launch_vm host1 rhel-9.3
+	prepare_kickstart host1 kickstart.ks.template rhel-9.3-microshift-crel-optionals
+	launch_vm host1
```

Comment:
You removed `rhel-9.3` from this line, but should you? I think it's installing el93 commit with el93 iso, but after removing it'll be el92 iso.
I think this should be reverted, in scenario.sh we need to change `DEFAULT_BOOT_BLUEPRINT`, and we need to add `rhel-9.2` in all scenarios that start with rhel92